cypress/integration/admin_portal/pages/enterprise_code_management.js

```diff
@@ -0,0 +1,33 @@
+class CodeManagementPage {
+  getFormField(field_name) {
+    return cy.get(`[name="${field_name}"]`)
+  }
+
+  getLabels(label_name) {
+    const labels = {'Email': 1, 'Company': 2, 'Number of Codes':3}
```

Comment:
you don't have to put '' around keys in dictionary unless there is space between words